release-notes/v1.1.0.md

```diff
@@ -20,7 +20,9 @@ The following table lists the known issues for HPE CSI Driver for Kubernetes v1.
 
 |ID|Component |Title|Description|Workaround|
 |--|---------|-----|-----------|----------|
+|CON-577|csi.k8s|Pods do not move to a new node when kubelet is unavailable|In case of node partition, (host unreachable) when the kubelet is unreachable the pods do not move to the new node. This is a known issue with kubernetes https://github.com/kubernetes/kubernetes/issues/65392 |1. Reboot the failed node. 2. force a move to another node after the NodeEviction Timeout has expired by entering the following command: kubectl delete pod <Pod_Name> --force --grace-period=0|
```

Comment:
Can you reverse the order. i.e force delete pods first and then reboot failed/partitioned node. This will ensure that we apply ACL to new node and remove from old node before rebooting failed one.

Reply:
makes sense. 👍